wordcloud/wordcloud_cli.py

```diff
@@ -20,10 +19,9 @@ def main(args):
         color_func=args.color_func, background_color=args.background_color).generate(args.text)
     image = wordcloud.to_image()
 
-    b = io.BytesIO()
-    image.save(b, format='png')
     with args.imagefile:
-        args.imagefile.write(b.getvalue())
+        out = args.imagefile if sys.version < '3' else args.imagefile.buffer
```

Comment:
It was/still is (see `args.imagefile = NamedTemporaryFile()`) in the diff above. Unfortunately it seems that `NamedTemporaryFile` behaves differently from actual files.
While I could test to see that this part is called just the way it is now that would not ensure that the functionality works. I don't see how that would help.